src/main/java/io/github/hypixel_api_wrapper/wrapper/util/GuildLevelingUtil.java

```diff
@@ -0,0 +1,159 @@
+package io.github.hypixel_api_wrapper.wrapper.util;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+// source code: https://github.com/HypixelDev/PublicAPI/blob/master/hypixel-api-core/src/main/java/net/hypixel/api/util/IGuildLeveling.java
+public class GuildLevelingUtil {
```

Comment:
Add a private constructor so that the class can't be instantiated, since it's just a helper

Reply:
Maybe inside that constructor throw an UnsupportedOperationException in case it's instantiated via reflection (for whatever reason)